pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -370,6 +374,28 @@ def _get_image_stream_name(assembly_type: assembly.AssemblyTypes, arch: str):
         go_arch_suffix = go_suffix_for_arch(arch)
         return f'4-dev-preview{go_arch_suffix}' if assembly_type == assembly.AssemblyTypes.PREVIEW else f'release{go_arch_suffix}'
 
+    async def sync_rhcos_srpms(self, assembly_type, data):
+        # Sync potential pre-release source on which RHCOS depends. See ART-6419 for details.
+        major, minor = util.isolate_major_minor_in_group(self.group)
+        if (major > 4 or (major == 4 and minor >= 13)) and assembly_type in [assembly.AssemblyTypes.CANDIDATE, assembly.AssemblyTypes.STANDARD]:
```

Comment:
Currently it is no longer needed for 4.13 and 4.14 since they have moved to 9.2 EUS. In case we need it in the future, maybe we need to turn it into a switch in ocp-build-data.